blog

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" type="text/css" href="styles.css">  
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  <title></title>
+</head>
+
+<body>
+
+<header>
+<div class="header-holder">
+<section class="left-side-header">
+<img src="images/adoptadog-logo.svg">
+<p>Seattle's premiere dog adoption agency</p>
+</section>
+<section class="right-side-header">
+  <p><i class="fas fa-shopping-cart"></i> $1234.56  </p>
+  <button> Checkout</button>
+    
+    </section>
+</div>    
+</header>
+<nav>
+<ul class="header-nav-list-holder">
+    <li class="header-nav-list-item"><a href="#home">Home</a></li>
+    <li class="header-nav-list-item">/</li>
+    <li class="header-nav-list-item"><a href="#dogs">Dogs</a></li>
+    <li class="header-nav-list-item">/</li>
+    <li class="header-nav-list-item"><a href="#blogs">Blogs</a></li>
+    <li class="header-nav-list-item">/</li>
+    </ul>
+    
+    </nav>    
+<!--<div id="wrapper">-->
+<h2 class="dogs-list-header">Here are all our Dogs!</h2>
+    <div class="grid-item-container"> 
+
+<section class="dog-holder">
+    <img class="adoption-pic" src="images/murphy-card.jpg">
+     <h3>Murphy</h3>       
+    <p class="dog-cost">Cost to Adopt: $123.45</p> 
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras maximus feugiat elementum. Pellentesque aliquet in lacus laoreet hendrerit. Proin eget tellus odio. Sed posuere sem eget blandit cursus. Curabitur venenatis, leo a malesuada sollicitudin, nunc diam luctus mauris, id semper urna massa maximus orci. </p>
+    <button class="adopt-button">Adopt</button>
+        </section>
+    <section class="dog-holder">
```

Comment:
Sections should be used for distinct sections on your page. Like maybe a section for your intro text, then a section for dog tiles. In this case, you'll want to wrap each dog tile with a div instead of a section.

Also, I would suggest adding padding to your dog-holder so the content isn't squashed in the box